include/seqan3/argument_parser/validators.hpp

```diff
@@ -322,17 +323,21 @@ class file_ext_validator // TODO AFTER INITIAL MERGE: Check if file exists, also
         extensions{v}
     {}
 
-    /*!\brief Tests whether cmp lies inside extensions.
-     * \param cmp The input value to check.
+    /*!\brief Tests whether p lies inside extensions.
```

Comment:
please call it path or filename. One character names are nasty to grep and replace later on :skull_and_crossbones: